client-app/src/examples/contact/mobile/DirectoryPanelModel.ts

```diff
@@ -72,10 +72,6 @@ export default class ContactsPageModel extends HoistModel {
         });
     }
 
-    updateContact(id: string, data: PlainObject) {
```

Comment:
Removing this in favour of refeshing AppModel.contacts after each change and then loading into the grid using the reaction above. This ensures the state is always consistent throughout the app.